docs/wiki/frameworks/custom-signal-func.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Custom radio signal strengh functions
```

Comment:
Should be the same as file name: `Custom Signal Functions` and rename file to `custom-signal-functions.md`.

I feel like a better name would be `Custom Signal Processing` though.